alloc.c

```diff
@@ -233,7 +233,7 @@ GC_API GC_stop_func GC_CALL GC_get_stop_func(void)
 
     if ((count++ & 3) != 0) return(0);
     GET_TIME(current_time);
-    time_diff = MS_TIME_DIFF(current_time,GC_start_time);
+    time_diff = NS_TIME_DIFF(current_time,GC_start_time);
```

Comment:
Is this the only usage we need to update?

Reply:
Yes. There are other uses, but those are all for logging/profiling outputs. I could change those as well, but figured that I'd rather do a more minimal change to keep us easier to merge with upstream.